src/scripts/main.js

```diff
@@ -1 +1,18 @@
 'use strict';
+
+const largeImg = document.getElementById('largeImg');
+const thumbnailList = document.getAnimations('thumbs');
```

Comment:
Why did you use `getAnimations` here?
You have to use `document.getElementById('thumbs')`. For now your Gallery doesn't work correctly.

Reply:
opps. its a typo. i wanted to type .getElementById ,